plugins/opa-backend/src/service/router.ts

```diff
@@ -52,15 +52,15 @@ export async function createRouter(
     }
   });
 
-  router.post('/catalog-permission', async (req, res) => {
-    const catalogPolicyInput = req.body.policyInput;
+  router.post('/opa-permissions', async (req, res) => {
```

Comment:
Still thinking about what to do here, I'd love to not have to specify the package in the `app-config.yaml` and then hardcode it here but cannot think of a way to do this yet.